libyul/optimiser/Suite.cpp

```diff
@@ -80,8 +81,16 @@ void OptimiserSuite::run(
 
 	NameDispenser dispenser{*_dialect, ast};
 
-	for (size_t i = 0; i < 4; i++)
+	size_t codeSize = 0;
```

Comment:
```suggestion
	size_t codeSize = -1U;
```
Just to be sure?

Reply:
I thought about that, but in the end - if the code size is already zero, what do you want to optimize?

Reply:
But if one happens to run the optimizer on code with zero code size, it will now run forever, won't it? Alternatively, we could also break not only for ``newSize == codeSize``, but also for ``newSize == 0`` below. Not that such cases are likely to happen, but maybe still?